src/components/Combobox.tsx

```diff
@@ -40,7 +40,7 @@ const defaultProps = {
 };
 
 const Combobox: React.FunctionComponent<ComboboxProps> = ({
-  className, direction, disabled, dropdownProps, options, placeholder, value, menuMaxHeight,
+  className, direction, disabled, dropdownProps, innerInputClassNames, options, placeholder, value, menuMaxHeight,
```

Comment:
Sorry to nitpick but `inputClassName`sounds a bit cleaner

Reply:
I agree, done!